### ISSUE TITLE
TypeError: $jscomp.initSymbol is not a function

### ISSUE THREAD

**Issue by glittershark** [thheller/shadow-cljs#736]
getting this on an upgrade to shadow-cljs 2.10
```
main.js:1671 TypeError: $jscomp.initSymbol is not a function
    at eval (vega-dataflow.js:48)
    at eval (vega-dataflow.js:3)
    at Object.shadow$provide.module$node_modules$vega_dataflow$build$vega_dataflow (vega-dataflow.js:2)
    at shadow.js.jsRequire (js.js:66)
    at Object.shadow$provide.module$node_modules$vega$build$vega_node (vega-node.js:7)
    at shadow.js.jsRequire (js.js:66)
    at Object.shadow$provide.module$node_modules$react_vega$lib$Vega (Vega.js:2)
    at shadow.js.jsRequire (js.js:66)
    at Object.shadow$provide.module$node_modules$react_vega$lib$index (index.js:2)
    at shadow.js.jsRequire (js.js:66)
```

**Comment by jiyinyiyong** [comment]
We may need some more details. Does your project just upgrade shadow-cljs from an older version, without cleaning `.shadow-cljs/`? How about `$jscomp`, is it defined? Is there any other errors in running your project that breaks initialization of `$jscomp`?

**Comment by thheller** [comment]
`2.10` is not a valid version. This should be fixed as of `2.10.9` but definitely had issues other `2.10.x`.

You can also set `:compiler-options {:output-feature-set :es6}` in your build config so it doesn't require those polyfills.
